build.gradle.kts

```diff
@@ -40,12 +40,21 @@ zapAddOn {
 
     manifest {
         author.set("KSASAN preetkaran20@gmail.com")
+
+        dependencies {
+            addOns {
+                register("commonlib")
```

Comment:
I didn't got this. Can you please help me understand the usage of this ?

Reply:
It establishes that the JWT add-on depends on the commonlib addon.

There are a number of pscan packages doing this now as well.

Reply:
ok. thanks for explaining, will read more on this.